docs/java-controller-tutorial-rewrite-rs-controller.md

```diff
@@ -0,0 +1,215 @@
+# Re-Write a Kubernetes Replicas-Controller in Java
+
+__TL;DR__: By this document, you will experience how to re-write a replicaset-controller
```

Comment:
nit: "This tutorial will show you how to re-write a ..."

Reply:
updated